zstacklib/zstacklib/test/utils/env.py

```
@@ -37,64 +58,146 @@ def value(self):
+    COVERAGE = env_var('coverage', bool, default=False, required=False).value()
 
 
 def log_env_variables():
-    logger.debug('environment variables: %s' % jsonobject.dumps(os.environ))
+    try:
+        if jsonobject is not None:
+            logger.debug('environment variables: %s' % jsonobject.dumps(os.environ))
+        else:
+            logger.debug('environment variables: %s' % dict(os.environ))
+    except Exception:
+        pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)